strongbox-db-import/src/main/resources/application.yml

```diff
@@ -1,23 +1,14 @@
 spring:
   main:
     web-application-type: NONE
-  liquibase:
-    change-log: classpath:/db/changelog/db.changelog-master.xml
 strongbox:
-  orientdb:
-    server:
-      protocol: remote
-      host: 127.0.0.1
-      port: "2025"
-      database: strongbox
-      username: admin
-      password: password
-      path: @strongbox.orientdb.path@
-    studio:
-      ipAddress: 127.0.0.1
-      port: 2480    
-      enabled: true
-      path: @strongbox.orientdb.path@
+  db:
+    janus-graph:
+      storage-username: cassandra
+      storage-password: cassandra
```

Comment:
I have been thinking that we need to make these passwords randomly generated upon first boot so that one can safely use Strongbox in production. Could you please add a `// TODO: [First start] randomize` here and before any other similar field(s) you come across? This will need to be planned as a separate follow-up task. :)

Reply:
done

Reply:
Thanks! :)

Reply:
We should probably have the default user/pass for this set to `admin/password`, as we're using this convention across the codebase already and we'll need to have a follow-up task to generate a password for this at the first boot. If we have different initial (default) credentials, it will make things more confusing.